File: android-sdk-ui/src/main/java/com/braze/ui/contentcards/recycler/ItemTouchHelperAdapter.java
package com.braze.ui.contentcards.recycler;

public interface ItemTouchHelperAdapter {
  void onItemDismiss(int position);

  /**
   * @param position The adapter position of the item.
   * @return True if the item at the adapter position is dismissable, false if the item is not dismissable.
   */
  boolean isItemDismissable(int position);
}

File: android-sdk-ui/src/main/java/com/braze/ui/contentcards/recycler/ItemTouchHelperAdapter.kt
package com.braze.ui.contentcards.recycler

interface ItemTouchHelperAdapter {
    fun onItemDismiss(position: Int)

    /**
     * @param position The adapter position of the item.
     * @return True if the item at the adapter position is dismissable, false if the item is not dismissable.
     */
    fun isItemDismissable(position: Int): Boolean
}
